### PR TITLE
Ghosts once again can see people's true names and roundstart jobs

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

#83186 made it so ghosts are fooled by disguises, like wearing a mask and an ID. This PR fixes that behavior, instead always displaying the person's real name and their face/ID name (if their face name is somehow different from real name) in brackets.

Additionally, this PR makes orbit menu prioritize "real" job name and icon, aka the ones the person spawned with. If they don't have an assigned job, it will fall back to current behavior of looking it up from their ID. Also, searching people includes both their fake and real name.

## Why It's Good For The Game

Ghosts really, really shouldn't be fooled by wearing a gas mask and an ID. **Especially** admin ghosts.

## Changelog
:cl:
qol: Ghost orbit menu now always displays person's real name and their roundstart job and cannot be fooled by disguises.
/:cl:

### CODE REVIEW

**Comment by jlsnow301** [comment]
you can literally just hover over the person to display their real name. this change was made because it makes the list tremendously space-inefficient.

File: tgui/packages/tgui/interfaces/Orbit/helpers.ts [hunk 1]
```diff
@@ -31,17 +31,7 @@ export function getDisplayName(full_name: string, nickname?: string): string {
...
-    !full_name?.includes('[') ||
-    full_name.match(/\(as /) ||
-    full_name.match(/^Unknown/)
-  ) {
-    return nickname;
-  }
-
-  // return only the name before the first ' [' or ' ('
-  return `"${full_name.split(/ \[| \(/)[0]}"`;
+  return nickname;
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/Orbit/helpers.ts:34]: you gutted this function to a point where it doesnt even need to exist, then didn't delete it despite it no longer being used

**Comment by jlsnow301** [comment]
image

This doesnt scale

**Comment by SmArtKar** [comment]
Real name/job display in main tab is now a toggle near the search bar in order to cut down on space used

**Comment by jlsnow301** [comment]
image

rems icon is broken here as the roundstart ai, the second one is an admin spawned unit. so, three things wrong with this picture imo.
1. ais not having icon
2. unknowns should not display an icon at all
3. if the user has no roundstart job but is wearing a trim, maybe they should display that?

**Pull request merged**